src/Knp/Menu/Twig/MenuExtension.php

```diff
@@ -121,4 +122,9 @@ public function isAncestor(ItemInterface $item, ?int $depth = null): bool
 
         return $this->runtimeExtension->isAncestor($item, $depth);
     }
+
+    public function spaceless(string $content): string
+    {
+        return trim((string) preg_replace('/>\s+</', '><', $content));
```

Comment:
https://github.com/twigphp/Twig/blob/4effb660b3eb6a2caa10fce0f88f06bdac0a4fd6/src/Extension/CoreExtension.php#L1233

Reply:
I don't see the reason for that. `$content` is already a string, so the null coalescing is unnecessary.  We need to cast the result instead (because preg_replace could return a null value), which is not done in Twig source code.

Reply:
it's fine. It's merely a reference to the original function for future readers.